src/dmd/dmodule.d

```diff
@@ -347,7 +347,7 @@ extern (C++) final class Module : Package
 
     const(char)[] arg;           // original argument name
     ModuleDeclaration* md;      // if !=null, the contents of the ModuleDeclaration declaration
-    const FileName srcfile;     // input source file
+    FileName srcfile;     // input source file
```

Comment:
It's not necessary to change the source file, as the user doesn't really see the preprocessed file and shouldn't. Preprocessing is something that happens to the contents of the srcfile along the way to being passed to the C parser.

Reply:
Probably true. It's currently wherever I shoved the logic when I first started working on this during the beerconf before last.